system-tests/tests/build.gradle.kts

```diff
@@ -35,10 +38,23 @@ dependencies {
     testImplementation(testFixtures(project(":common:util")))
     testImplementation(testFixtures(project(":launchers:junit")))
 
+    testImplementation("com.linecorp.armeria:armeria-grpc-protocol:${armeriaVersion}")
```

Comment:
I'm no expert on the Armeria Server, but could we achieve the same thing with `mock-webserver`? Does it *have to* use gRPC? We already have `mock-webserver` as a dependency in other tests, and I'd like to avoid pulling in multiple libs that achieve the same purpose.

Reply:
@paullatzelsperger I applied your suggestion. I used mock server instead of armeria and configured the java agent to use http instead of gRPC.